cmd/singularity/security_test.go

```diff
@@ -91,7 +91,7 @@ func testSecurityPriv(t *testing.T) {
 
 // testSecurityConfOwnership tests checks on config files ownerships
 func testSecurityConfOwnership(t *testing.T) {
-	configFile := files.GetSysConfigFile()
+	configFile := syfs.GetSysConfigFile()
```

Comment:
Same here. Don't really know if a function call is intended, but `buildcfg.SINGULARITY_CONF_FILE` can be used

Reply:
@sashayakovtseva FYI, that was part of the initial work: the code was full of `buildcfg.SINGULARITY_CONF_FILE = "string"` and at one spot it was not correctly used at all. So a function seems more useful.